packages/data/addon/mirage/handlers/graphql.js

```diff
@@ -129,6 +377,20 @@ function _getResponseBody(db, parent) {
         )
       );
 
+      // handle limit in request
+      if (first < rows.length) {
```

Comment:
Can first be null? I think in request v2 we decided that limit is `number|null`

Reply:
yeah when it's null this evaluates to false and no limit is applied

Reply:
Oh actually `first` will be undefined instead of null because when the request has a null limit, it just doesn't include it in the query. `undefined < ${number}` will always be false

Reply:
👍 gotcha, worth having a test though